packages/webviz-core/src/panels/ThreeDimensionalViz/SceneBuilder/index.js

```diff
@@ -796,6 +798,25 @@ export default class SceneBuilder implements MarkerProvider {
         // flatten btn: set empty z values to be at the same level as the flattenedZHeightPose
         this._consumeOccupancyGrid(topic, deepParse(message));
         break;
+      case NAV_MSGS_PATH_DATATYPE: {
+        const pathStamped = cast<BinaryPath>(message);
+        if (pathStamped.poses().length() === 0) {
+          break;
+        }
+        const newMessage = {
+          header: deepParse(pathStamped.header()),
+          // TODO(@cjds) Make this make use of the orientation of the poses in the path as well
+          points: pathStamped
+            .poses()
+            .toArray()
+            .map((pose) => deepParse(pose.pose().position())),
+          closed: false,
+          scale: { x: 0.2 },
+          color: { r: 1, g: 0, b: 0, a: 1 },
```

Comment:
Would be nice to add `overrideColor` topic settings for setting r,g,b,a. Similar to rviz: http://wiki.ros.org/rviz/DisplayTypes/Path

Reply:
That would be awesome. Will change it. I copied this from `PolygonStamped` .. Was there any reason that didn't use the overrides?
https://github.com/cruise-automation/webviz/blob/eab0da004a6bc6e90a9107d1c46a0df3337cf70e/packages/webviz-core/src/panels/ThreeDimensionalViz/SceneBuilder/index.js#L838

Reply:
![wv2](https://user-images.githubusercontent.com/4957157/111385920-b5d09a80-8668-11eb-926a-54afed1d0298.gif)
Got the override working